react-ui-kit/src/Inputs/Suggest/index.tsx

```diff
@@ -77,10 +78,12 @@ export const InputSuggest: FC<IInputSuggest> = ({
     return list.map((label, index) => ({ label, value: index }));
   }, [list]);
 
-  const menuOpen = useMemo((): boolean => {
-    const selected = list.some((label) => valueInput === label);
-    return focused && !!valueInput && valueInput.length >= charsToShowSuggest && !selected;
-  }, [focused, valueInput, list, charsToShowSuggest]);
+  const menuOpen = (): boolean | undefined => {
```

Comment:
Функцию можно не делать
```suggestion
  const menuOpen = charsToShowSuggest >= valueInput.length || !valueInput || !focused ? false : undefined;
```
А в чём принципиальная разница между false и undefined (и то и то выступает как ложь)? Почему нет значения true? Меню никогда не бывает открыто?